src/metrics/runner.py

```diff
@@ -301,7 +301,11 @@ def _create_runner_stop(
         repo=runner_metrics.pre_job.repository,
         github_event=runner_metrics.pre_job.event,
         status=runner_metrics.post_job.status,
-        status_info=runner_metrics.post_job.status_info,
+        status_info=(
+            runner_metrics.post_job.status_info.dict()
```

Comment:
Is it no longer a dict?

Reply:
It was and still a `CodeInformation` class but in Pydantic v1 it did not complain. In Pydantic v2 it started to complain and fail integration tests.